mods/ra/maps/fort-lonestar/fort-lonestar.lua

```diff
@@ -197,9 +197,13 @@ end
 WorldLoaded = function()
 	soviets = Player.GetPlayer("Soviets")
 	players = { }
-	for i = 0, 4, 1 do
+	for i = 0, 4 do
 		local player = Player.GetPlayer("Multi" ..i)
 		players[i] = player
+
+		if players[i] and players[i].Name == "Lonestar AI" then
```

Comment:
I agree.  Checking against `IsBot` should be the minimum (because a real player may set their name to this), but if there are multiple AI types then checking `IsBot` and also the `Name` would be okay.